### ISSUE TITLE
How to compile Aegisub

### ISSUE THREAD

**Issue by Nikit4v** [Aegisub/Aegisub#286]
I am trying to compile Aegisub from source since May.
I tried to build it on manjaro linux, ubuntu, fedora, windows 10 (via msys), windows 11 (via mvs and msys), macos big sur and macos monterey.
There is always something wrong.

On vs I usually have hundreds of syntax errors and such.
On linux and mac it's usually something like linking errors.
Where is the complete "how to compile Aegisub" guide?
(help me please...)

**Comment by moex3** [comment]
Yes, this is a problem. On windows, I had success with the wangqr fork [here](https://github.com/wangqr/Aegisub#cmake-for-windows-linux-and-macos). On linux, this repo or the typesettingtools one generally worked. You can get some help from the [gentoo build script](https://gitweb.gentoo.org/repo/gentoo.git/tree/media-video/aegisub/aegisub-9999.ebuild).

But what is exactly the problem, on linux for example. Can you post the build logs?

**Comment by Nikit4v** [comment]
I specifically installed Linux to compile Aegisub, but I will not use it. It would be great if you could help with compiling on macOS. I am attaching the logs of everything in a [zip archive](https://github.com/Aegisub/Aegisub/files/9250913/logs.zip).

**Comment by FichteFoll** [comment]
I've been building and running the TypesettingTools fork for a long time now, ever since it initially got meson support. I also maintain the corresponding package on the AUR. Unfortunately, it doesn't build currently because of an issue with meson that you can read more about at https://github.com/TypesettingTools/Aegisub/issues/162.

Other than that, is there a specific reason you couldn't use one of the pre-built packages such as for your Linux distro's package manager or one of the TypesettingTools' CI builds for macOS and Windows: https://github.com/TypesettingTools/Aegisub/actions/runs/2603641658

**Comment by darealshinji** [comment]
Here are build scripts and a Linux Appimage in the release section: https://github.com/darealshinji/aegisub-appimage
Maybe that will help. Someone could use them to create nightly builds if they want.

Too bad that there are so many open PRs, even simple ones with important build process fixes. Is the original project dead? aegisub.org seems to be offline too.

**Comment by Nikit4v** [comment]
>  Is the original project dead? aegisub.org seems to be offline too.

Seems like that. Good time to create a new editor XD.

> Here are build scripts and a Linux Appimage in the release section: https://github.com/darealshinji/aegisub-appimage
Maybe that will help. 

Thank you. I install linux (again) and this script works. I'll read it later.  I think if i will understand this script i could compile aegisub in other operation systems (i hope).
